scan.py

```diff
@@ -3,7 +3,8 @@
 
 
 def show_aggie_pride():
-    slogan_list = ['Aggie Pride - Worldwide']
+    slogan_list = ['Aggie Pride - Worldwide',
+                    'And That\'s On My 1891']
```

Comment:
This looks fine, however, I wanted to point out that python allows you another way to do this.
```
"And That's On My 1891" 
```
Would work as well.  No real right or wrong way, just personal preference